pkg/cable/ipsec/getconnections.go

```diff
@@ -0,0 +1,88 @@
+package ipsec
+
+import (
+	"github.com/bronze1man/goStrongswanVici"
+	"k8s.io/klog"
+
+	"github.com/submariner-io/submariner/pkg/cable"
+)
+
+func (i *strongSwan) GetConnections() (*[]cable.Connection, error) {
+
+	client, err := getClient()
+	if err != nil {
+		return nil, err
+	}
+	defer client.Close()
+
+	sas, err := client.ListSas("", "")
+	if err != nil {
+		klog.Errorf("Error while retrieving active IKE_SAs during connection list: %v", err)
+		return nil, err
```

Comment:
```suggestion
		return nil, errors.WithMessage(err, "Error while retrieving active IKE_SAs")
```

Also no need to also log here.

Reply:
Nice I didn't know that errros.WithMessage :-)